### DISCUSSION TITLE
Disabling multiple arrows to the same node

### DISCUSSION THREAD

**Discussion by aquaslvt** [mermaid-js/mermaid#5148] [Q&A]
Hello! How do you make it so that, automatically, when there's the same node connecting to another node  multiple times it just displays one arrow?

For example, when I type
```
A --> B
A --> B
A --> B
```

It displays:

```
[A]
/|\
\|/
[B]
```

But I want it to automatically display:

```
[A]
 |
[B]
```

**Comment by huynhicode** [comment]
Hi @mkukiro - What is the use case for wanting to display the same node connected to another node multiple times?  We might be able to write different syntax.
